src/Controller/User/ChangePasswordController.php

```diff
@@ -43,7 +43,7 @@ public function __construct(UserPasswordEncoderInterface $passwordEncoder)
     public function changePasswordAction(Request $request)
     {
         $dto = new ChangePasswordDTO();
-        $form = $this->createForm(ChangePasswordType::class, $dto);
+        $form = $this->createForm(ChangePasswordType::class, $dto, ['validation_groups' => ['registration']]);
```

Comment:
Why are we using a `registration` group here?  I'm not sure why we would need any validation group.
- Note that using a group named`registration` does not make sense here as the form is for changing a password and has nothing to do with registering a user.

Leaving it off also means no changes would need to be made to `Form/Type/ChangePasswordType.php` instead of adding a validation group there as well, and `Form/DTO/ChangePasswordDTO.php ` would also not have a validation group specifiied in the three locations it is now (by this PR) being specified there.